data/migrations/20211215202038_Submissions_Table.js

```diff
@@ -0,0 +1,23 @@
+exports.up = function (knex) {
+  return knex.schema.alterTable('Submissions-New', (t) => {
+    t.timestamp('startAt');
+    t.timestamp('readAt');
+    t.timestamp('drawAt');
+    t.timestamp('writeAt');
+    t.timestamp('squadUpAt');
+    t.timestamp('voteAt');
+    t.string('gameMode').notNullable();
+  });
+};
+
+exports.down = function (knex) {
+  return knex.schema.alterTable('Submissions-New', (t) => {
```

Comment:
alter Submissions table instead of Submissions-New.

Reply:
Thanks Chris, I'll make these changes shortly.